frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/version/manager/VersionManagerType.java

```diff
@@ -0,0 +1,22 @@
+package com.github.eirslett.maven.plugins.frontend.lib.version.manager;
+
+public enum VersionManagerType {
+    FNM("fnm"),
+    NVM("nvm");
+
+    // TODO add support for other version managers
+    //    MISE("mise"),
+    //    ASDF("asdf"),
+    //    NVS("nvs"),
```

Comment:
Yeah, I think we're going to need these for this to get adoption

Reply:
I've added nvs, still asdf and mise left

Reply:
Added mise, asdf left

Reply:
asdf added